.travis.yml

```diff
@@ -10,6 +10,10 @@ virtualenv:
   system_site_packages: true
 
 before_install:
+<<<<<<< HEAD
+  - pip install mpmath
```

Comment:
In general, when you do a merge, make sure you pay attention to the output - if Git asks you to manually merge, make sure you've actually resolved all conflicts! Don't do a `git add -a`.